flink-clients/src/test/java/org/apache/flink/client/program/ClientTest.java

```diff
@@ -304,4 +368,60 @@ public String getDescription() {
 			return "TestOptimizerPlan <input-file-path> <output-file-path>";
 		}
 	}
+
+	public static final class TestExecuteTwice {
+
+		public static void main(String args[]) throws Exception {
```

Comment:
A `PackagedProgram` needs to have a main class with a main method with a `public` modifier. That's why it wouldn't work.